src/graph.rs

```diff
@@ -3019,8 +3022,6 @@ struct PendingState<'a> {
   pending: FuturesOrdered<PendingInfoFuture<'a>>,
   jsr: PendingJsrState,
   npm: PendingNpmState,
-  pending_specifiers:
-    HashMap<ModuleSpecifier, HashSet<Option<AttributeTypeWithRange>>>,
```

Comment:
What it was doing here was quite broken handling of attribute types. The change I made is also broken, but slightly better. To fix this properly we need to key on attribute type in addition to specifier for pending loads, but that's a big change for the future.

Reply:
It's not really handled by V8 - given (specifier, attributes) it is embedders responsibility to provide the proper source for the module.

Reply:
Oh, ok, I'll do a follow-up to fix. Before it was actually only sometimes erroring.

Reply:
Back in the wmr days we serialized the import attributes to query params and added that to the file URL. That worked quite well and allowed us to keep the cache logic flat